udemy_exercises/Sorting.java

```diff
@@ -1,25 +1,54 @@
 package udemy_exercises;
 
+import java.util.*;
+
 class Sorting {
 
+    int numArray[] = new int[10];
+
     void sortIntegers(int[] nums) {
 
+        Arrays.sort(nums);
+        System.out.println(Arrays.toString(reverse(nums)));
     }
 
     void printNums() {
-
+        System.out.println(Arrays.toString(getNums()));
     }
 
     int[] getNums() {
-        return new int[]{};
+
+        Scanner s = new Scanner(System.in);
+
+        System.out.println("Please enter 10 numbers");
+        for (int i = 0; i < 10; i++) {
+            numArray[i] = s.nextInt();
+        }
+
+        s.close();
+
+        return numArray;
     }
 
-    void reverse(int[] numbers) {
+    int[] reverse(int[] numbers) {
+
+        int tempArr[] = new int[10];
+        int count = 0;
+
+        for (int i = 9; i >= 0; i--) {
+            tempArr[count] = numbers[i];
+            count++;
+        }
```

Comment:
over-engineered. `for (int i=0; i<10; i++) { 
tempArr[i] = numbers[numbers.length - 1 - i];
}`